website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -70,20 +70,22 @@ This POST endpoint requires a JSON object with the following properties as a req
+| `data.attributes.vcs-repo.branch`             | string         | `null`    | The branch of the VCS repo. If empty, the VCS provider's default branch value will be used.                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `data.attributes.vcs-repo.identifier`         | string         |           | The identifier of the VCS repository in the format `<namespace>/<repo>`. For example, on GitHub, this would be something like `hashicorp/my-policy-set`. The format for Azure DevOps is `<org>/<project>/_git/<repo>`.                                                                                                                                                                                                                                                                                                                             |
+| `data.attributes.vcs-repo.oauth-token-id`     | string         |           | The OAuth Token ID to use to connect to the VCS host.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `data.attributes.vcs-repo.ingress-submodules` | boolean        | `false`   | Determines whether repository submodules will be instantiated during the clone operation.                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
The clone operation felt....very technical. Let me know if I misinterpreted what that means!

```suggestion
| `data.attributes.vcs-repo.ingress-submodules` | boolean        | `false`   | Whether Terraform Cloud should instantiate repository submodules when retrieving the policy set.                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```